testing/ef_tests/src/cases/epoch_processing.rs

```diff
@@ -201,6 +201,7 @@ impl<E: EthSpec> EpochTransition<E> for PendingBalanceDeposits {
     }
 }
 
+// TODO(alloy) test fails here
```

Comment:
This comment can be removed as the tests pass now

Reply:
deleted TODO